utils/biospecimen.js

```diff
@@ -673,11 +673,12 @@ const biospecimenAPIs = async (req, res) => {
         }
         const supplyQuery = req.query.supplyKitId;
         const collectionQuery = (req.query.collectionId?.slice(0, -4) || "") + " " + (req.query.collectionId?.slice(-4) || ""); // add space to collection
+        const returnKitTrackingNumberQuery = req.query.returnKitTrackingNumber;
```

Comment:
It's a tracking number, not a query, right?

Reply:
It's the tracking number pulled from the query. I'm following the naming convention from supplyQuery and collectionQuery, but I'm happy to change it if you'd rather.

Reply:
I don't think those names should be followed. They should be updated instead for clarity.